include/picongpu/particles/creation/moduleInterfaces/SkipSuperCellFunctor.hpp

```diff
@@ -0,0 +1,36 @@
+/* Copyright 2024 Brian Marre
+ *
+ * This file is part of PIConGPU.
+ *
+ * PIConGPU is free software you can redistribute it and or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * PIConGPU is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with PIConGPU.
+ * If not, see <http://www.gnu.org/licenses/>.
+ */
+
+#pragma once
+
+#include "picongpu/simulation_defines.hpp"
+
+namespace picongpu::particles::creation::moduleInterfaces
+{
+    //! interface of SkipTestFunctor
```

Comment:
What is a skip test functor? IMO `test` should not be in the name.

It should also have a description with the motivation. e.g.
```
Depending of the supercellindex and the input parameter a supercell can be excluded from the creation process.
```

Reply:
Done